optaplanner-core/src/main/java/org/optaplanner/core/api/package-info.java

```diff
@@ -18,7 +18,8 @@
  * The public API of OptaPlanner.
  * <p>
  * All classes in this namespace are backwards compatible in future releases (especially minor and hotfix releases).
- * If a major version number changes, a few specific classes might not be backwards compatible,
- * in which case the upgrade recipe will clearly document those cases.
+ * If a major version number changes, a few specific classes might not be backwards compatible, in which case
+ * <a href="http://www.optaplanner.org/download/upgradeRecipe/upgradeRecipeArchive.html">the upgrade recipe</a>
```

Comment:
Maybe we should rename upgradeRecipeArchive.html to index.html (same for releaseNotesArchive.html), so then we can use a shorter URL here: href="http://www.optaplanner.org/download/upgradeRecipe/"
and it's likely that some users will craft that url themselfes to find a list of all upgrade recipe's (when looking at 1 upgrade recipe) - in which case they'd currently see "forbidden".